### PR TITLE
CAZ-1234 Get and update payment status

### PR DESCRIPTION
**Pull request by radekmorytko**

### CODE REVIEW

File: src/main/resources/db/changelog/changes/0001-1.0-create_inital_schemas_with_audit_log.yaml [hunk 1]
```diff
@@ -220,7 +231,7 @@ databaseChangeLog:
                   constraints:
                     nullable: false
   - changeSet:
-      id: 0001.3-1.1
+      id: 0001.4-1.1
```
Comment (jaroslawpiotrowski) [inline src/main/resources/db/changelog/changes/0001-1.0-create_inital_schemas_with_audit_log.yaml]: Shouldn't be `0001.4-1.0`?

File: src/main/java/uk/gov/caz/psr/dto/GetAndUpdatePaymentStatusResponse.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+   */
+  public static GetAndUpdatePaymentStatusResponse from(Payment payment) {
+    return new GetAndUpdatePaymentStatusResponse(payment.getId(),
+        fromInternalStatus(payment.getStatus()));
+  }
+
+  /**
+   * Maps the internal payment's status to the one returned to the REST API caller.
+   */
+  private static PaymentStatus fromInternalStatus(uk.gov.caz.psr.model.PaymentStatus status) {
```
Comment (jaroslawpiotrowski) [inline src/main/java/uk/gov/caz/psr/dto/GetAndUpdatePaymentStatusResponse.java]: Can you fix import?

File: src/main/resources/db/changelog/changes/0001-1.0-create_inital_schemas_with_audit_log.yaml (hunk 1 above)
Comment (radekmorytko) [reply src/main/resources/db/changelog/changes/0001-1.0-create_inital_schemas_with_audit_log.yaml]: Good catch, fixed.

File: src/main/java/uk/gov/caz/psr/dto/GetAndUpdatePaymentStatusResponse.java (hunk 2 above)
Comment (radekmorytko) [reply src/main/java/uk/gov/caz/psr/dto/GetAndUpdatePaymentStatusResponse.java]: I renamed the class so now there are no clashes with names, thanks.

**Pull request merged**